### ISSUE TITLE
QR Code

### ISSUE THREAD

**Issue by jcvernaleo** [decred/Paymetheus#298]
New Address page should provide a QR code of the address (assuming a suitable library exists).

**Comment by jcvernaleo** [comment]
@jrick if Paymetheus is really dead, should this one get closed?

**Comment by jrick** [comment]
Yes, we could close/lock/archive all of the issues.  Not sure the best way to go about that.

**Comment by jcvernaleo** [comment]
Does archiving the project do anything useful with old issues?  I haven't ever tried it so I don't know what it really does.

**Comment by jrick** [comment]
Not sure.  Is there any particular reason why you care about these old open issues?  We can try to accommodate for that.

I think there's a way we can make the whole project read-only, but not sure if that is the best solution.

**Comment by jcvernaleo** [comment]
I'm just trying to clear out the gigantic mess of old issues I have listed in my github account to try and prevent issues I care about from slipping through the cracks so occasionally I've been trying to follow up on old ones.  It isn't vital of course, I just like to keep things neat when I can.
